### PR TITLE
Add bindings for Adwaita/libadwaita

### PR DESCRIPTION
**Pull request by sideeffffect**
Inspired by
 * https://news.ycombinator.com/item?id=39844936
 * https://www.swift.org/blog/adwaita-swift/

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -63,6 +64,29 @@ lazy val examples = project
...
+  .configure(pkgConfigured("libadwaita-1"))
+  .settings(
+    bindgenBindings +=
+      buildWithDependencies(
+        "gtk",
+        "gio",
+        "glib",
+        "gobject",
+        "libcairo",
+        "libharfbuzz"
```
Comment (keynmol) [inline build.sbt]: I think you also need graphene, pango, gsk, and gdk

File: build.sbt [hunk 2]
```diff
@@ -63,6 +64,32 @@ lazy val examples = project
...
+        "libharfbuzz",
+        "pango"
+      ) {
+        Binding
+          .builder(findHeader("libadwaita-1", _ / "adwaita.h"), "adwaita")
+          .withClangFlags(pkgConfig("libadwaita-1", "cflags"))
+          .addCImport("adwaita.h")
+          .withOpaqueStructs(Set("G*", "Adw*"))
+          .withMultiFile(true)
+          .addExternalPath("*/gdk-pixbuf-2.0/*", "libgdkpixbuf")
```
Comment (keynmol) [inline build.sbt]: can you move this to `buildWithDependencies` please?

File: build.sbt (hunk 1 above)
Comment (sideeffffect) [reply build.sbt]: thanks

File: build.sbt (hunk 2 above)
Comment (sideeffffect) [reply build.sbt]: done

**Comment by keynmol** [comment]
Awesome! thank you

**Pull request merged**